lib/src/components/form/chip_input.dart

```diff
@@ -87,7 +96,6 @@ class ChipInputState<T> extends State<ChipInput<T>> with FormValueSupplier {
       final theme = Theme.of(context);
       _popoverController.show(
         context: context,
-        handler: const PopoverOverlayHandler(),
```

Comment:
I think this might cause an issue on mobile since mobile replaces popover handler with bottom sheet,
this was added to override the mobile behavior and force the suggestion box to show popover instead of bottom sheet.
If this causes a problem, then removing this might not be the solution.

Reply:
It was removed, by mistake. I will add it back